libavcodec/x86/vvc/vvc_lmcs.asm

```diff
@@ -0,0 +1,255 @@
+%include "libavutil/x86/x86util.asm"
+
+SECTION_RODATA 32
+
+pb_shufmask_16bpc:   db 0, 1, 4, 5, 8, 9, 12, 13, -1, -1, -1, -1, -1, -1, -1, -1, 0, 1, 4, 5, 8, 9, 12, 13, -1, -1, -1, -1, -1, -1, -1, -1
+pb_shufmask_8bpc:  db 0, 4, 8, 12, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 0, 4, 8, 12, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1
+
+SECTION .text
+
+; Only handles widths 16 to 128
+INIT_YMM avx2
+
+cglobal lmcs_16bpc, 5, 8, 13, src, src_stride, width, height, lut, src_pt, cnt, src2
+    pxor             m0, m0 ; 0 register
+    movu             m12, [pb_shufmask_16bpc]
+
+    .loopheight:
+        mov             src_ptq, srcq ; saveptr
+        lea             src2q, [srcq + src_strideq]

```

Comment:
It will be better to align all of your `,`. You can check other asm files on how the alignment works.